CONTRIBUTING.md

```diff
@@ -1,81 +1,104 @@
-   The recipe creation/update/bug-fix is one of our major development items, and we really encourage 
-   you to work on it.
-   When you port a Kaldi recipe to ESPnet, see https://github.com/espnet/espnet/wiki/How-to-port-the-Kaldi-recipe-to-the-ESPnet-recipe%3F 
+ESPnet1 follows convention from [Kaldi](https://github.com/kaldi-asr/kaldi), and it is also based on several utilities from Kaldi. ESPnet1's recipes
+correspond to `egs`. This feature naturally makes it very simple to port a Kaldi recipe to ESPnet. If you port a Kaldi recipe to ESPnet, please see
+[port-kaldi-recipe](https://github.com/espnet/espnet/wiki/How-to-port-the-Kaldi-recipe-to-the-ESPnet-recipe%3F) for more detailed instructions.
+If there is no existing Kaldi's recipe, you could still refer to 
+[port-kaldi-recipe](https://github.com/espnet/espnet/wiki/How-to-port-the-Kaldi-recipe-to-the-ESPnet-recipe%3F) and the major task is to prepare
+a Kaldi-style directory (Please see [Prepare-Kaldi-Style-Directory](https://kaldi-asr.org/doc/data_prep.html) for details.)
    
-   We also encourage you to report your results with your detailed environmental info and upload the model for the reproducibility 
-   (e.g., see https://github.com/espnet/espnet/blob/master/egs/tedlium2/asr1/RESULTS.md).
+For each recipe, we also encourage you to report your results with your detailed environmental info and upload the model for the reproducibility 
+(e.g., see [tedlium2-example](https://github.com/espnet/espnet/blob/master/egs/tedlium2/asr1/RESULTS.md)).
    
-   To make a report for `RESULTS.md`
-	 - execute `show_result.sh` at a recipe main directory (where `run.sh` is located), as follows. 
-	   You'll get environmental information and the evaluation result of each experiments in a markdown format.
-	   ```
-	   $ show_result.sh
-	   ```
-	 - execute `pack_model.sh` at a recipe main directory as follows. You'll get model information in a markdown format
-	   ```
-	   $ pack_model.sh --lm <language model> <tr_conf> <dec_conf> <cmvn> <e2e>
-	   ```
-	 - `pack_model.sh` also produces a packed espnet model (`model.tar.gz`). If you upload this model to somewhere with a download link,
-	   please put the link information to `RESULTS.md`.
-	 - please contact Shinji Watanabe <shinjiw@ieee.org> if you want a web storage to put your model files.
-
-Once you finish implementing a feature or bugfix, please send a Pull Request to https://github.com/espnet/espnet
+To make a report for `RESULTS.md`
+ - execute `show_result.sh` at a recipe main directory (where `run.sh` is located), as follows. 
+   You'll get environmental information and the evaluation result of each experiment in a markdown format.
+   ```
+   $ show_result.sh
+   ```
+ - execute `pack_model.sh` at the main directory of a recipe as follows. You'll get model information in a markdown format
+   ```
+   $ pack_model.sh --lm <language model> <tr_conf> <dec_conf> <cmvn> <e2e>
```

Comment:
```suggestion
   $ pack_model.sh --lm <language model> --dict <dict> <tr_conf> <dec_conf> <cmvn> <e2e>
```

Also, not an issue but I was wondering: shouldn't we have a parameter for outputted tar or parse the training config name? There are a lot of configs in some recipe (e.g.: commonvoic or vivos) and we need to manually rename after each packing